src/subscriber.ts

```diff
@@ -94,36 +81,51 @@ export class Message {
      * @name Message#ackId
```

Comment:
For the constructor, can we use a real interface instead of the inline type?

Reply:
Sure!